instrumentation/netty/netty-4.1/javaagent/src/test/groovy/Netty41ClientTest.groovy

```diff
@@ -40,134 +33,18 @@ import java.util.concurrent.TimeUnit
 import static org.junit.jupiter.api.Assumptions.assumeTrue
 
 @Unroll
-class Netty41ClientTest extends HttpClientTest<DefaultFullHttpRequest> implements AgentTestTrait {
```

Comment:
I extracted as much of this to a `AbstractNetty41ClientTest` as I could but certain bits were tightly intwined with groove to easily pull apart.